channel_log.go

```diff
@@ -45,7 +45,7 @@ func sanitizeBody(body string) string {
 	ct := httpx.DetectContentType([]byte(parts[1]))
 
 	// if this isn't text, replace with placeholder
-	if !strings.HasPrefix(ct, "text") {
+	if !strings.HasPrefix(ct, "text") && ct != "application/json" {
```

Comment:
maybe better to use HasPrefix in case it gives us something like `application/json; charset=utf-8`